tasmota/tasmota_support/support_command.ino

```diff
@@ -329,6 +329,7 @@ void ExecuteCommand(const char *cmnd, uint32_t source)
   // cmnd: "var1=1"    = stopic "var1" and svalue "=1"
   SHOW_FREE_MEM(PSTR("ExecuteCommand"));
   ShowSource(source);
+  TasmotaGlobal.last_source = source;
```

Comment:
This breaks previous use of ``last_source`` (as used by ``ExecuteWebcommand()``

Do NOT do it here but in your application.

Reply:
hmm..  can you suggest how to get the last_source where we need it?  Maybe this was a patch which fixed lack of correctly set last_source in v10 at the right place, and now the source is set correctly in v12?

Reply:
If you need last_source in your code, copy the ExecuteCommand functionality like the ExecuteWebCommand() in your code like:

```
void ExecuteTuyaCommand(char* svalue, uint32_t source) {
  TasmotaGlobal.last_source = source;
  ExecuteCommand(svalue, source);
}
```

and change your calls to ``ExecuteCommand()`` into ``ExecuteTuyaCommand()``. This doesn't break other code using the last_source in a different way as you do.